nixos/modules/base-modules.nix

```
@@ -0,0 +1,187 @@
+[
```

This could be a module instead.

```suggestion
{ imports = [
```